<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_ShareLinks</class>
 <widget class="QWidget" name="DB_ShareLinks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="DB_CustomMessageBox" name="messageBox" native="true">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="DB_CustomFrame_3" name="frame_loading">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_loading{
border: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_telephone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>153</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(':/images/courseware/telephone.png');</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_telephone">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>145</y>
     <width>245</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(165, 165, 165);</string>
   </property>
  </widget>
  <widget class="DB_CustomLabel_1" name="label_telephoneShare">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>145</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78,155,252);
color: white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>分享至账号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_link">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(':/images/courseware/link.png');</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_link">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>195</y>
     <width>245</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(165, 165, 165);</string>
   </property>
  </widget>
  <widget class="DB_CustomLabel_1" name="label_copyLink">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>195</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78,155,252);
color: white;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>复制链接</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_title">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_title{
border: none;
background-color: rgb(78,155,252);
border-top-left-radius: 8px;
border-top-right-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>分享课件</string>
    </property>
   </widget>
   <widget class="DB_CustomLabel_1" name="label_close">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;:/images/courseware/close-3.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>491</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(':/images/file-1.png');</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 85);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DB_CustomLabel_1</class>
   <extends>QLabel</extends>
   <header location="global">CustomWidget/db_customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>DB_CustomFrame_3</class>
   <extends>QFrame</extends>
   <header location="global">CustomWidget/db_customframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DB_CustomMessageBox</class>
   <extends>QWidget</extends>
   <header location="global">CustomWidget/db_custommessagebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
